src/transports/mod.rs

```diff
@@ -15,6 +15,9 @@ pub mod ws;
 #[cfg(feature = "ws")]
 pub use self::ws::WebSocket;
 
+#[cfg(feature = "test")]
```

Comment:
```suggestion
#[cfg(any(feature = "test", test))]
```

Reply:
Can you explain what this does?

Reply:
It enables the module either when we run tests for `web3` or when `test` feature for the crate is enabled.